src/components/layout/ConfirmRegistration/ConfirmRegistration.tsx

```diff
@@ -0,0 +1,119 @@
+import { useState } from 'react';
+
+import { useGlobalState } from '../../../state/contexts/GlobalState';
+import { useRegistrationState } from '../../../state/contexts/RegistrationState';
+import { NAME_PRICE_INFO } from '../../../utils/constants';
+import { AlertCircle } from '../../icons';
+import './styles.css';
+
+function ConfirmRegistration() {
+  const [{ walletAddress }] = useGlobalState();
+  const [
+    {
+      domain,
+      chosenTier,
+      ticker,
+      nickname,
+      controllers,
+      ttl,
+      leaseDuration,
+      targetID,
+      antID,
+      fee,
+      stage,
+    },
+    dispatchRegistrationState,
+  ] = useRegistrationState();
+  const [priceInfo, setPriceInfo] = useState(false);
+
+  return (
+    <>
+      <div className="register-name-modal center">
```

Comment:
nit: add a `TODO` to update ant-card and use it here

Reply:
todo added